presto-geospatial/src/main/java/io/prestosql/plugin/geospatial/GeoFunctions.java

```diff
@@ -1341,6 +1365,33 @@ private static OGCGeometry geomFromBinary(Slice input)
         return geometry;
     }
 
+    private static OGCGeometry geomFromHadoopShape(Slice input)
+    {
+        requireNonNull(input, "input is null");
```

Comment:
This isn't needed for an internal helper method, since we know it won't be called with null.